dask_expr/expr.py

```diff
@@ -853,6 +853,13 @@ class DropnaFrame(Blockwise):
     operation = M.dropna
 
 
+class Fillna(Blockwise):
+    _parameters = ["frame", "value", "method", "limit", "axis"]
+    _defaults = {"value": None, "method": None, "limit": None, "axis": None}
+    _keyword_only = ["method", "limit", "axis"]
+    operation = M.fillna
```

Comment:
Unfortunately this won't work here (at least not in all cases). We can not inherit from ``Blockwise``. Fillna is implemented using ``map_overlap`` in dask/dask, which is necessary for properly filling NaN values when method is set and one block finished with ``NaN``. That's one of the reasons I skipped it for now.

Reply:
Ah. I should have known that things are not so simple, or you would have implemented it already.

Do you have a reproduction case for when it would not work?

Reply:
Something like:

```
df = pd.DataFrame({"a": [1, 2, np.nan, 3]})
ddf = from_pandas(df, npartitions=2)
ddf.fillna(method="ffill")
```

should reproduce. First row from one of the partitions should be missing and you have to specify a method that relies on the previous values.